doc/manual-wiki/misc.wiki

```diff
@@ -47,55 +47,62 @@ To simplify the creation of <<span class="code"|<link>>> tags
 
 ===@@id="basic_menu"@@Basic menus
 
-
-      To build a menu on your Web page, you can use the function
-          <<a_api| val Eliom_tools.menu >>.
-      First, define your menu like this:
-
-
-
-
-<<code language="ocaml"|let mymenu current =
-  Eliom_tools.menu ~classe:["menuprincipal"]
-    (home, %:xmllist< Home ~>>)
-    [
-     (infos, %:xmllist< More info ~>>);
-     (tutorial, %:xmllist< Documentation ~>>)
-   ] current
+To generate a context-aware menu on your Web page, you can use the function
+<<a_api| val Eliom_tools.HTML5_TOOLS.menu>>. This function can be used from
+either of the modules <<span class="teletype"|Eliom_tools.Html.D>> or
+<<span class="teletype"|Eliom_tools.Html.F>>. (See
+<<a_manual chapter="clientserver-html#unique"|
+HTML element manipulation, by value and by reference>>).
+
+Here is a simple example:
+
+<<code language="ocaml"|let mymenu =
+  let items =
+    [(home,     [pcdata "Home"]);
+     (info,     [pcdata "More info"]);
+     (tutorial, [pcdata "Documentation"])]
+  in
+    Eliom_tools.D.menu
+    ~classe:["menuprincipal"]
+    items
+    ~service:Eliom_service.reload_action
 >>
 
+<<span class="teletype"|items>> is a list of pairs correlating the services to
+be linked with the text to be displayed: <<span class="teletype"|home>>,
+<<span class="teletype"|info>>, and <<span class="teletype"|tutorial>> are our
+three services (generated, for example, by
+<<a_api| val Eliom_service.App.create>> or
+<<a_api| val Eliom_service.Html.regiser>>).
```

Comment:
Do you mean `Eliom_registration.{App,Html}`?

Reply:
I did! I failed to do proper research here again, and just modeled this after the after the existing documentation. Thank you for catching this!